docs/topics/items.rst

```diff
@@ -42,7 +42,8 @@ Item objects
 :class:`Item` provides a :class:`dict`-like API plus additional features that
 make it the most feature-complete item type:
 
-.. class:: Item([arg])
+.. class:: scrapy.item.Item
+.. class:: scrapy.Item([arg])
```

Comment:
When not using `autoclass`, the signature needs to be defined manually:

```suggestion
.. class:: scrapy.item.Item([arg])
.. class:: scrapy.Item([arg])
```

Same applies to other, similar changes below.

Reply:
Thank you, I will fix it!